.docker/test.ini

```diff
@@ -31,13 +17,9 @@ full_stack = true
 cache_dir = /tmp/%(ckan.site_id)s/
 beaker.session.key = ckan
```

Comment:
beaker.session.key is replaced with SECRET_KEY from the looks of the doc's

Reply:
Actually it's `beaker.session.secret` that's replaced. But only in 2.11+ so we can't drop it yet.